### PR TITLE
[Selenium] Adapt selenium tests from dashboard package

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Rewrite next tests from **dashboard** package:
- WorkspacesListTest
- WorkspaceDetailsProjectsTest
- RenameWorkspaceTest

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13827

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java [hunk 1]
```diff
@@ -209,89 +176,37 @@ public void checkWorkspaceSelectingByCheckbox() throws Exception {
...
+    workspaces.typeToSearchInput(sequenceForSearch);
 
-    // check reverse order after "RAM" clicking
-    workspaces.clickOnRamButton();
-    items = workspaces.getVisibleWorkspaces();
     try {
-      assertEquals(items.get(0).getRamAmount(), JAVA_WS_MB);
-      assertEquals(items.get(1).getRamAmount(), BLANK_WS_MB);
-    } catch (AssertionError ex) {
+      workspaces.waitVisibleWorkspacesCount(EXPECTED_SORTED_WORKSPACES_COUNT);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java]: Question about name `EXPECTED_SORTED_WORKSPACES_COUNT` - I can't find statement to sort projects. 
Does it mean "EXPECTED_FOUND_WORKSPACES_COUNT`?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java [hunk 2]
```diff
@@ -349,45 +263,20 @@ public void checkWorkspaceActions() throws Exception {
...
-    List<Workspaces.WorkspaceListItem> items = workspaces.getVisibleWorkspaces();
-
-    Workspaces.WorkspaceListItem currentDisplayingJavaItem =
-        workspaces.getWorkspacesListItemByWorkspaceName(items, expectedJavaItem.getWorkspaceName());
-
-    assertEquals(currentDisplayingJavaItem, expectedJavaItem);
+    workspaces.waitWorkspaceIsNotPresent(NEWEST_CREATED_WORKSPACE_NAME);
   }
 
   private int getWorkspacesCount() throws Exception {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java]: It duplicates `TestWorkspaceServiceClient::getWorkspacesCount()` and so can be removed.

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Review by dmytro-ndp — APPROVED** [review]

**Review by artaleks9 — APPROVED** [review]

**Pull request merged**